src/jobs/characters.js

```diff
@@ -0,0 +1,68 @@
+const CronJob = require('cron').CronJob;
+import moment from 'moment';
+import models from '../models';
+import { Op } from 'sequelize';
+
+const Character = models.Character;
+
+async function updateCharacter(character) {
+  await Character.update({
+    battlegroup: null,
+    faction: null,
+    class: null,
+    race: null,
+    gender: null,
+    achievementPoints: null,
+    thumbnail: null,
+    spec: null,
+    role: null,
+    talents: null,
+    heartOfAzeroth: null
```

Comment:
I think updatedAt should be set here as well

Reply:
Yup, that'd solve both this comment and the previous one

Reply:
My other concern would be making sure that in modules/character, if we "find" the character, we make sure that it has the important data in it before returning the json. dont want to send a bunch of nulls back to the analyzer! probably just checking for class or spec exists would be enough?